<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="mainWidget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="MTitleWidget" name="titleWidget" native="true">
           <property name="toolTip">
            <string>标题栏</string>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MenuWidget" name="menuWidget" native="true"/>
         </item>
         <item>
          <widget class="ToolWidget" name="toolWidget" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="contentWidget" native="true"/>
         </item>
         <item>
          <widget class="StackedWidget" name="stackedWidget">
           <widget class="QWidget" name="page"/>
           <widget class="QWidget" name="page_2"/>
          </widget>
         </item>
        </layout>
        <zorder>titleWidget</zorder>
        <zorder>menuWidget</zorder>
        <zorder>contentWidget</zorder>
        <zorder>toolWidget</zorder>
        <zorder>stackedWidget</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MTitleWidget</class>
   <extends>QWidget</extends>
   <header>Ui/Library/MTitleWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MenuWidget</class>
   <extends>QWidget</extends>
   <header>Ui/Library/MenuWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolWidget</class>
   <extends>QWidget</extends>
   <header>Ui/Library/ToolWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>Ui/Library/Stackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../FCS.qrc"/>
 </resources>
 <connections/>
</ui>
